cryptofeed/exchange/binance.py

```diff
@@ -310,7 +321,65 @@ async def _funding(self, msg: dict, timestamp: float):
                             rate=msg['r'],
                             next_funding_time=timestamp_normalize(self.id, msg['T']),
                             )
+    
+    async def _funding_with_index_price(self, msg: dict, timestamp: float):
+        """
+        {
+            "e": "markPriceUpdate",     // Event type
+            "E": 1562305380000,         // Event time
+            "s": "BTCUSDT",             // Symbol
+            "p": "11794.15000000",      // Mark price
+            "i": "11784.62659091",      // Index price
+            "P": "11784.25641265",      // Estimated Settle Price, only useful in the last hour before the settlement starts
+            "r": "0.00038167",          // Funding rate
+            "T": 1562306400000          // Next funding time
+        }
+        """
+        symbol = self.exchange_symbol_to_std_symbol(msg['s'])
+        index_symbol = self.exchange_symbol_to_std_symbol('.' + msg['s'])
+        ts = timestamp_normalize(self.id, msg['E'])
+
+        if msg['s'] in self.subscription[feed_to_exchange(self.id, FUNDING)]:
```

Comment:
We have to check that we have actually subscribed to the symbol in the message. Otherwise we would stream INDEX events for the future, and FUNDING events for the index.

Reply:
This is a bit confusing and probs needs some helper functions.

Reply:
Shoud show more obviously that we try to publish an event for both if we're subbed